torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py

```diff
@@ -105,39 +97,35 @@ def forward_call(self, *inputs, **kwargs):
                     self._export_mode,
                 )
 
-        try:
-           # Create the inference_session
+        # Create the inference_session
+        if not self._inference_session:
            session_options, providers, provider_options = self._get_session_config()
            self._inference_session = onnxruntime.InferenceSession(
-                self._onnx_models.exported_model.SerializeToString() , session_options, providers, provider_options
-            )
-
-           run_options = C.RunOptions()
-
-           # pre-process inputs to make them compatible with onnxruntime
-           io_binding = self._inference_session.io_binding()
-           # Use IO binding
-           ort_inputs = []
-           for inp in inputs:
-               ort_inputs.append(inp)
-           _utils._create_iobinding(io_binding, inputs, self._onnx_models.exported_model, self._device)
-           self._inference_session.run_with_iobinding(io_binding, run_options)
-           forward_outputs = io_binding.get_outputs()
-
-           # forward outputs is a list (std::vector<OrtValue>) but _ortvalues_to_torch_tensor
-           # is expected a OrtValueVector (also std::vector<OrtValue> but defined in onnxruntime-training).
+            self._onnx_models.exported_model.SerializeToString() , session_options, providers, provider_options
+           )
+
+        run_options = C.RunOptions()
+
+        # Pre-process inputs to make them compatible with onnxruntime
+        
+        # Use IO binding
+        io_binding = self._inference_session.io_binding()
+        _utils._create_iobinding(io_binding, inputs, self._onnx_models.exported_model, self._device)
+        
+        # Run inference session
+        self._inference_session.run_with_iobinding(io_binding, run_options)
```

Comment:
Do we need a session with io_binding? Does this provide any performance/latency benefits?

Reply:
we would still need io_binding since the inputs are converted from torch tensors to ortvalues.